poc/vdaf_mastic.py

```diff
@@ -339,7 +341,7 @@ def joint_rand_seed(cls, joint_rand_parts):
         return cls.Xof.derive_seed(
             zeros(cls.Xof.SEED_SIZE),
             cls.domain_separation_tag(USAGE_JOINT_RAND_SEED),
-            concat(joint_rand_parts),
+            concat(joint_rand_parts)
```

Comment:
Why make this change? Did the linter suggest this?

In Python3 I believe the convention is to terminate every item in a list with a comma, even the last. (This is the convention in more modern programming languages.)

Reply:
Picked the wrong branch in the merge. Fixed now.